src/apps/content/actions/product-actions.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Dependencies.
+ */
+
+let AppDispatcher = require('../dispatcher/app-dispatcher');
```

Comment:
lowlevel stuff:

we are now using `-` `_` and `camelCase` for js
which one is the "right" one ? 

cc @RStankov @korbin